Changelog.md

```diff
@@ -365,7 +365,7 @@ concurrently using this.
 * Add `takeWhileM` and `dropWhileM`
 * Add `filterM`
 
-## 0.3.0
+## 0.3.0 - 19th June 2018
```

Comment:
15th June according to https://github.com/composewell/streamly/releases

Reply:
Hackage says 19th June, (maybe docs were uploaded manually and it uses that as the uploaded date?)